### DISCUSSION TITLE
[20 Marzo 2023] - Cache

### DISCUSSION THREAD

**Discussion by alem1105** [CS-Swap/Architettura-degli-Elaboratori#16] [Cache]
### Traccia Esercizio

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/622ecf03-b387-46cf-a726-8f48bddd2507)

<details><summary>Soluzione Prof</summary>
<p>

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/5a50e43e-f56d-4a98-b336-6baa8ac0f451)

</p>
</details>

**Comment by notedo** [comment]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/1d0542c9-c7ec-4ffa-8ef4-71dee5a6bd40)
